requirements.txt

```diff
@@ -1 +1 @@
-nailgun
+git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
```

Comment:
iek, should we just wait until a proper nailgun release with your changes is out?